### PR TITLE
Added Meet Page

### PR DESCRIPTION
**Pull request by baylee-rl**
Added the beginning of the meet page w/ CSS from the Figma

### CODE REVIEW

File: src/pages/Meet.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+.meetContainer h1 {
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 108px;
+    line-height: 67px;
+/* identical to box height */
+    text-align: center;
+    color: #000000;
+}
```
Comment (TheWillMundy) [inline src/pages/Meet.css]: image
Running into this issue with the default zoom on my browser, adjust the line-height to solve this

**Review by TheWillMundy — CHANGES_REQUESTED** [review]
Wow. Fantastic work! Just a single change to make, once that's done this is perfect to merge.

File: src/pages/Meet.css (hunk 1 above)
Comment (baylee-rl) [reply src/pages/Meet.css]: Thank you! The formatting issues should be fixed now

**Review by TheWillMundy — APPROVED** [review]
Looks great! I added a quick change in the `.memberBoxContainer` to ensure mobile responsiveness (changed `grid-template-columns` from `repeat(3, 1fr)` to `repeat(auto-fit, minmax(300px, 1fr))` and removed the padding as it's no longer necessary)
Otherwise everything looks great!

**Pull request merged**